code/game/machinery/telecomms/broadcaster.dm

```diff
@@ -158,6 +158,24 @@ var/message_delay = 0 // To make sure restarting the recentmessages list is kept
 
 
 
+/proc/Is_Bad_Connection(var/old_freq, var/new_freq) //Makes sure players cant read radios of a higher level than they are
```

Comment:
`var/` in arguments is implied, so you can just make this `/proc/Is_Bad_Connection(old_freq, new_freq)`